README.md

```diff
@@ -3,16 +3,27 @@
 
 # Symphony Workflow Developer Kit (WDK)
 
-_Also known as Workflow API or Workflow bot_, check out the  [documentation](./docs).
+The Symphony Workflow Developer Kit (WDK) is a way to build bots on the Symphony platform with minimal coding efforts. A
+standard execution engine, provided as a bot is running workflows written in a declarative manner.
+
+Workflow executions are triggered by events (such as Datafeed events) and run activities. Activities are small building
+blocks often calling REST API endpoints of the Symphony platform or can even be custom-made to tailor your needs.
+
+- [Getting started](./getting-started.md): a beginner's guide to run your first workflow
+- [Concepts](./concepts.md): high-level view of the key concepts behind workflows
+- [Architecture](./architecture.md): technical bits on the execution of workflows
+- [Deployment](./deployment.md): how to run and configure the workflow bot
+- [Custom activities](./custom-activities.md): step-by-step guide on how to implement your own activities
+- [Examples](./examples): concrete and advanced workflows
+- [SWADL Reference](./reference.md): syntax reference for the language used to write workflows
+
 
 :warning: _The WDK is very much in its inception and we are actually looking to get feedback on its usage. It also means
 all the APIs and the SWADL format are subject to change._
-
-## Getting started
-Check out the [Getting starting guide](./docs/getting-started.md) for an introduction.
+<hr />
 
 ## Usage example
-
+![](./gifs/wdk-demo.gif)
```

Comment:
I'd prefer to move the `./gifs` folder inside `./docs` to let project root as light as possible